lib/omniship/dhlgm/track/package.rb

```diff
@@ -58,10 +58,29 @@ def has_left?
           return false
         end
 
+        # 510 : SHIPMENT ACCEPTED BY USPS
+        # 520 : ARRIVAL AT POST OFFICE
+        # 538 : DEPART USPS SORT FACILITY
+        # has been accepted by the post office or has been accepted any of these should be good
+        def has_arrived_at_usps?
```

Comment:
this should be private since its not in line with the other public functions on the other package classes. They all match.